package.json

```diff
@@ -38,9 +39,9 @@
     "eslint-plugin-sql-template": "^2.0.0"
   },
   "devDependencies": {
-    "@uphold/github-changelog-generator": "^1.0.1",
-    "eslint": "^7.18.0",
-    "mocha": "^7.1.1",
+    "@uphold/github-changelog-generator": "^3.0.0",
+    "eslint": "^8.7.0",
```

Comment:
Update on the `peerDependencies` as well?

Reply:
should we set peerDependencies to be greater than >8.7.0 ?

Reply:
Yeah good spot - my test projects never required this but I'll bump this up too.